Index: bindings/c/CMakeLists.txt
===================================================================
--- bindings/c/CMakeLists.txt	(revision 1052478)
+++ bindings/c/CMakeLists.txt	(working copy)
@@ -57,4 +57,4 @@
 ########### install files ###############
 
 INSTALL( FILES  ${CMAKE_CURRENT_BINARY_DIR}/taglib_c.pc DESTINATION ${LIB_INSTALL_DIR}/pkgconfig)
-INSTALL( FILES  tag_c.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  tag_c.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: bindings/c/taglib_c.pc.cmake
===================================================================
--- bindings/c/taglib_c.pc.cmake	(revision 1052478)
+++ bindings/c/taglib_c.pc.cmake	(working copy)
@@ -9,4 +9,4 @@
 Requires: taglib
 Version: ${TAGLIB_LIB_MAJOR_VERSION}.${TAGLIB_LIB_MINOR_VERSION}.${TAGLIB_LIB_PATCH_VERSION}
 Libs: -L${LIB_INSTALL_DIR} -ltag_c
-Cflags: -I${INCLUDE_INSTALL_DIR}/taglib 
+Cflags: -I${INCLUDE_INSTALL_DIR} 
Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 1052478)
+++ CMakeLists.txt	(working copy)
@@ -2,14 +2,14 @@
 
 cmake_minimum_required(VERSION 2.6.0 FATAL_ERROR)
 
-OPTION(ENABLE_STATIC "Make static version of libtag"  OFF)
+OPTION(ENABLE_STATIC "Make static version of libtag"  ON)
 
 OPTION(BUILD_TESTS "Build the test suite"  OFF)
 OPTION(BUILD_EXAMPLES "Build the examples"  OFF)
 
 OPTION(NO_ITUNES_HACKS "Disable workarounds for iTunes bugs"  OFF)
-OPTION(WITH_ASF "Enable ASF tag reading/writing code"  OFF)
-OPTION(WITH_MP4 "Enable MP4 tag reading/writing code"  OFF)
+OPTION(WITH_ASF "Enable ASF tag reading/writing code"  ON)
+OPTION(WITH_MP4 "Enable MP4 tag reading/writing code"  ON)
 
 add_definitions(-DHAVE_CONFIG_H)
 
@@ -62,7 +62,7 @@
     set(TAGLIB_WITH_MP4 TRUE)
 endif(WITH_MP4)
 configure_file(taglib/taglib_config.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/taglib_config.h )
-install( FILES  ${CMAKE_CURRENT_BINARY_DIR}/taglib_config.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+install( FILES  ${CMAKE_CURRENT_BINARY_DIR}/taglib_config.h DESTINATION ${INCLUDE_INSTALL_DIR})
 
 ADD_SUBDIRECTORY( taglib ) 
 
Index: taglib.pc.cmake
===================================================================
--- taglib.pc.cmake	(revision 1052478)
+++ taglib.pc.cmake	(working copy)
@@ -8,4 +8,4 @@
 Requires: 
 Version: ${TAGLIB_LIB_MAJOR_VERSION}.${TAGLIB_LIB_MINOR_VERSION}.${TAGLIB_LIB_PATCH_VERSION}
 Libs: -L${LIB_INSTALL_DIR} -ltag
-Cflags: -I${INCLUDE_INSTALL_DIR}/taglib 
+Cflags: -I${INCLUDE_INSTALL_DIR} 
Index: taglib/ape/CMakeLists.txt
===================================================================
--- taglib/ape/CMakeLists.txt	(revision 1052478)
+++ taglib/ape/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  apetag.h apefooter.h apeitem.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  apetag.h apefooter.h apeitem.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/asf/CMakeLists.txt
===================================================================
--- taglib/asf/CMakeLists.txt	(revision 1052478)
+++ taglib/asf/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  asffile.h asfproperties.h asftag.h asfattribute.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  asffile.h asfproperties.h asftag.h asfattribute.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/CMakeLists.txt
===================================================================
--- taglib/CMakeLists.txt	(revision 1052478)
+++ taglib/CMakeLists.txt	(working copy)
@@ -209,4 +209,4 @@
 	ARCHIVE DESTINATION  ${LIB_INSTALL_DIR}
 )
 
-INSTALL( FILES  tag.h fileref.h audioproperties.h taglib_export.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  tag.h fileref.h audioproperties.h taglib_export.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/flac/CMakeLists.txt
===================================================================
--- taglib/flac/CMakeLists.txt	(revision 1052478)
+++ taglib/flac/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  flacfile.h flacproperties.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib )
+INSTALL( FILES  flacfile.h flacproperties.h DESTINATION ${INCLUDE_INSTALL_DIR} )
Index: taglib/mp4/CMakeLists.txt
===================================================================
--- taglib/mp4/CMakeLists.txt	(revision 1052478)
+++ taglib/mp4/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  mp4file.h mp4atom.h mp4tag.h mp4item.h mp4properties.h mp4coverart.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  mp4file.h mp4atom.h mp4tag.h mp4item.h mp4properties.h mp4coverart.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/mpc/CMakeLists.txt
===================================================================
--- taglib/mpc/CMakeLists.txt	(revision 1052478)
+++ taglib/mpc/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  mpcfile.h mpcproperties.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  mpcfile.h mpcproperties.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/mpeg/CMakeLists.txt
===================================================================
--- taglib/mpeg/CMakeLists.txt	(revision 1052478)
+++ taglib/mpeg/CMakeLists.txt	(working copy)
@@ -1,4 +1,4 @@
 ADD_SUBDIRECTORY( id3v1 ) 
 ADD_SUBDIRECTORY( id3v2 ) 
 
-INSTALL(FILES  mpegfile.h mpegproperties.h mpegheader.h xingheader.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib )
+INSTALL(FILES  mpegfile.h mpegproperties.h mpegheader.h xingheader.h DESTINATION ${INCLUDE_INSTALL_DIR} )
Index: taglib/mpeg/id3v1/CMakeLists.txt
===================================================================
--- taglib/mpeg/id3v1/CMakeLists.txt	(revision 1052478)
+++ taglib/mpeg/id3v1/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  id3v1tag.h id3v1genres.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  id3v1tag.h id3v1genres.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/mpeg/id3v2/CMakeLists.txt
===================================================================
--- taglib/mpeg/id3v2/CMakeLists.txt	(revision 1052478)
+++ taglib/mpeg/id3v2/CMakeLists.txt	(working copy)
@@ -1,4 +1,4 @@
 ADD_SUBDIRECTORY( frames ) 
 
-INSTALL(FILES  id3v2extendedheader.h id3v2frame.h id3v2header.h  	id3v2synchdata.h id3v2footer.h id3v2framefactory.h id3v2tag.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL(FILES  id3v2extendedheader.h id3v2frame.h id3v2header.h  	id3v2synchdata.h id3v2footer.h id3v2framefactory.h id3v2tag.h DESTINATION ${INCLUDE_INSTALL_DIR})
 
Index: taglib/mpeg/id3v2/frames/CMakeLists.txt
===================================================================
--- taglib/mpeg/id3v2/frames/CMakeLists.txt	(revision 1052478)
+++ taglib/mpeg/id3v2/frames/CMakeLists.txt	(working copy)
@@ -10,4 +10,4 @@
 	unknownframe.h
 	unsynchronizedlyricsframe.h
 	urllinkframe.h
-	DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+	DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/ogg/CMakeLists.txt
===================================================================
--- taglib/ogg/CMakeLists.txt	(revision 1052478)
+++ taglib/ogg/CMakeLists.txt	(working copy)
@@ -2,5 +2,5 @@
 ADD_SUBDIRECTORY( speex ) 
 ADD_SUBDIRECTORY( flac ) 
 
-INSTALL( FILES  oggfile.h  	oggpage.h  	oggpageheader.h  	xiphcomment.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib )
+INSTALL( FILES  oggfile.h  	oggpage.h  	oggpageheader.h  	xiphcomment.h DESTINATION ${INCLUDE_INSTALL_DIR} )
 
Index: taglib/ogg/flac/CMakeLists.txt
===================================================================
--- taglib/ogg/flac/CMakeLists.txt	(revision 1052478)
+++ taglib/ogg/flac/CMakeLists.txt	(working copy)
@@ -1,2 +1,2 @@
-INSTALL( FILES  oggflacfile.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  oggflacfile.h DESTINATION ${INCLUDE_INSTALL_DIR})
 
Index: taglib/ogg/speex/CMakeLists.txt
===================================================================
--- taglib/ogg/speex/CMakeLists.txt	(revision 1052478)
+++ taglib/ogg/speex/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  speexfile.h speexproperties.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  speexfile.h speexproperties.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/ogg/vorbis/CMakeLists.txt
===================================================================
--- taglib/ogg/vorbis/CMakeLists.txt	(revision 1052478)
+++ taglib/ogg/vorbis/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  vorbisfile.h vorbisproperties.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  vorbisfile.h vorbisproperties.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/riff/aiff/CMakeLists.txt
===================================================================
--- taglib/riff/aiff/CMakeLists.txt	(revision 1052478)
+++ taglib/riff/aiff/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  aifffile.h aiffproperties.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  aifffile.h aiffproperties.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/riff/CMakeLists.txt
===================================================================
--- taglib/riff/CMakeLists.txt	(revision 1052478)
+++ taglib/riff/CMakeLists.txt	(working copy)
@@ -1,4 +1,4 @@
 ADD_SUBDIRECTORY( aiff )
 ADD_SUBDIRECTORY( wav )
 
-INSTALL( FILES  rifffile.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  rifffile.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/riff/wav/CMakeLists.txt
===================================================================
--- taglib/riff/wav/CMakeLists.txt	(revision 1052478)
+++ taglib/riff/wav/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  wavfile.h wavproperties.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  wavfile.h wavproperties.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/toolkit/CMakeLists.txt
===================================================================
--- taglib/toolkit/CMakeLists.txt	(revision 1052478)
+++ taglib/toolkit/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  taglib.h tstring.h tlist.h tlist.tcc tstringlist.h  	tbytevector.h tbytevectorlist.h tfile.h  	tmap.h tmap.tcc DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  taglib.h tstring.h tlist.h tlist.tcc tstringlist.h  	tbytevector.h tbytevectorlist.h tfile.h  	tmap.h tmap.tcc DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/trueaudio/CMakeLists.txt
===================================================================
--- taglib/trueaudio/CMakeLists.txt	(revision 1052478)
+++ taglib/trueaudio/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  trueaudiofile.h trueaudioproperties.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  trueaudiofile.h trueaudioproperties.h DESTINATION ${INCLUDE_INSTALL_DIR})
Index: taglib/wavpack/CMakeLists.txt
===================================================================
--- taglib/wavpack/CMakeLists.txt	(revision 1052478)
+++ taglib/wavpack/CMakeLists.txt	(working copy)
@@ -1 +1 @@
-INSTALL( FILES  wavpackfile.h wavpackproperties.h DESTINATION ${INCLUDE_INSTALL_DIR}/taglib)
+INSTALL( FILES  wavpackfile.h wavpackproperties.h DESTINATION ${INCLUDE_INSTALL_DIR})
